src/test/java/org/tarantool/jdbc/JdbcResultSetIT.java

```diff
@@ -370,4 +371,70 @@ public void testTraversal() throws SQLException {
         assertEquals(0, resultSet.getRow());
     }
 
+    @Test
+    public void testMaxFieldSize() throws SQLException {
+        assertEquals(0, stmt.getMaxFieldSize());
+
+        int expectedMaxSize = 256;
+        stmt.setMaxFieldSize(expectedMaxSize);
+        assertEquals(expectedMaxSize, stmt.getMaxFieldSize());
+    }
+
+    @Test
+    public void testNegativeMaxFieldSize() throws SQLException {
+        SQLException error = assertThrows(SQLException.class, () -> stmt.setMaxFieldSize(-12));
+        assertEquals(SQLStates.INVALID_PARAMETER_VALUE.getSqlState(), error.getSQLState());
+    }
+
+    @Test
+    public void testPositiveMaxFieldSize() throws SQLException {
+        testHelper.executeSql("INSERT INTO test VALUES (1, 'greater-than-ten-characters-value')");
+
+        stmt.setMaxFieldSize(10);
+        try (ResultSet resultSet = stmt.executeQuery("SELECT * FROM test WHERE id = 1")) {
+            resultSet.next();
+            assertEquals("greater-th", resultSet.getString(2));
+        }
+    }
+
+    @Test
+    public void testMaxFieldSizeBiggerThanValue() throws SQLException {
+        testHelper.executeSql("INSERT INTO test VALUES (1, 'less-than-one-hundred-characters-value')");
+
+        stmt.setMaxFieldSize(100);
+        try (ResultSet resultSet = stmt.executeQuery("SELECT * FROM test WHERE id = 1")) {
+            resultSet.next();
+            assertEquals("less-than-one-hundred-characters-value", resultSet.getString(2));
+        }
+    }
+
+    @Test
+    public void testMaxFieldSizeBinaryType() throws SQLException {
+        testHelper.executeSql("CREATE TABLE test_bin(id INT PRIMARY KEY, val SCALAR)");
+        testHelper.executeSql("INSERT INTO test_bin VALUES (1, 'binary-string')");
```

Comment:
It will be a inserted as mp_str, while we want to test mp_bin value. It can be inserted as X'62696e6172792d737472696e67'. Or maybe better to add this test when we'll implement #214. Now it looks as duplicate of `testPositiveMaxFieldSize()`.